impeller/entity/contents/atlas_contents.cc

```diff
@@ -57,6 +64,96 @@ void AtlasContents::SetCullRect(std::optional<Rect> cull_rect) {
   cull_rect_ = cull_rect;
 }
 
+struct AtlasBlenderKey {
+  Color color;
+  Rect rect;
+  int32_t color_key;
+
+  static int32_t ToColorKey(Color color) {
+    return (((std::lround(color.alpha * 255) & 0xff) << 24) |
```

Comment:
Ah, the `kEhCloseEnough` of the color world. Perhaps we should have an `IColor` in `color.h` that does this for us?